File: swiperefresh/src/main/java/com/gu/swiperefresh/Utils/Log.java
package com.gu.swiperefresh.Utils;

/**
 * Created by gu on 2016/12/23.
 */

public class Log {
    private static final boolean DEBUG = false;

    public static void i(String tag, String info) {
        if (DEBUG)
            android.util.Log.i(tag, info);
    }

    public static void d(String tag, String info) {
        if (DEBUG)
            android.util.Log.d(tag, info);
    }

    public static void e(String tag, String info) {
        if (DEBUG)
            android.util.Log.e(tag, info);
    }

    public static void v(String tag, String info) {
        if (DEBUG)
            android.util.Log.v(tag, info);
    }
}

File: swiperefresh/src/main/java/me/guhy/swiperefresh/Utils/Log.kt
package me.guhy.swiperefresh.Utils

/**
 * Created by gu on 2016/12/23.
 */

object Log {
    private val DEBUG = false

    @JvmStatic fun i(tag: String, info: String) {
        if (DEBUG) {
            android.util.Log.i(tag, info)
        }
    }

    @JvmStatic fun d(tag: String, info: String) {
        if (DEBUG) {
            android.util.Log.d(tag, info)
        }
    }

    @JvmStatic fun e(tag: String, info: String) {
        if (DEBUG) {
            android.util.Log.e(tag, info)
        }
    }

    @JvmStatic fun v(tag: String, info: String) {
        if (DEBUG) {
            android.util.Log.v(tag, info)
        }
    }
}
